src/pages/TestK8s.tsx

```diff
@@ -1,18 +1,16 @@
 /* eslint-disable no-console */
 import * as React from 'react';
-import { Button, PageSection, TextInput } from '@patternfly/react-core';
 import {
+  k8sListResourceItems,
+  k8sGetResource,
   k8sCreateResource,
   k8sDeleteResource,
-  k8sGetResource,
-  k8sListResource,
-  K8sModel,
   k8sPatchResource,
-  K8sResourceCommon,
   k8sUpdateResource,
-  // useK8sWatchResource,
-  // WatchK8sResource,
-} from '../dynamic-plugin-sdk';
+  // K8sResourceCommon,
+} from '@openshift/dynamic-plugin-sdk-utils';
+import { Button, PageSection, TextInput } from '@patternfly/react-core';
+import { K8sModel, K8sResourceCommon } from '../dynamic-plugin-sdk';
```

Comment:
These are available now.
```suggestion
import { K8sModelCommon, K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
```